articles/defender-for-cloud/quickstart-onboard-aws.md

```diff
@@ -67,7 +67,7 @@ The native cloud connector requires:
         > To enable the Azure Arc auto-provisioning, you'll need **Owner** permission on the relevant Azure subscription.
         
     - Additional extensions should be enabled on the Arc-connected machines.
-    - Log Analytics (LA) agent on Arc machines, and ensure the selected workspace has security solution installed. The LA agent is currently configured in the subscription level. All of your multicloud AWS accounts and GCP projects under the same subscription will inherit the subscription settings.
+    - Log Analytics (LA) agent on Arc machines, or Azure Monitor agent (AMA), and ensure the selected workspace has security solution installed. The LA agent and AMA are currently configured in the subscription level. All of your multicloud AWS accounts and GCP projects under the same subscription will inherit the subscription settings.
```

Comment:
This sentence looks like it's missing something. What is the customer supposed to do with "Log Analytics (LA) agent on Arc machines, or Azure Monitor agent (AMA),"